setup.py

```diff
@@ -26,5 +34,5 @@
         'Topic :: Scientific/Engineering',
     ],
     license='apache',
-    packages=setuptools.find_packages()
```

Comment:
this change can be reverted if we don't specify symbols to import...

Reply:
the importing style should not cause much problem i think.